<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmmain</class>
 <widget class="QMainWindow" name="frmmain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CEMSTech AVRxpal</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txtterminal">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Droid Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot; bgcolor=&quot;#000000&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpterminal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="bttnterminal">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Terminal/cmd...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../home/shovon/Desktop/python stuffs/gui/Terminal-icon.png</normaloff>../home/shovon/Desktop/python stuffs/gui/Terminal-icon.png</iconset>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdoprgfolder">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>in app dir.</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdohexfolder">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>116</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>in hex dir.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpprgset">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial Port Settings</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Select port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Baud            :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboport">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cbobaudrate">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bttnrefreshports">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>30</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblnote1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Note:Above settings are only
for Arduino and serial port based
programmers.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblprg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>341</width>
      <height>17</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpfile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Select file to load</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="bttnbrowse">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../home/shovon/Desktop/python stuffs/gui/Open.png</normaloff>../home/shovon/Desktop/python stuffs/gui/Open.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtfilename">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>291</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdoflash">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Flash</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdoeeprom">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>EEPROM</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bttnwrite">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../home/shovon/Desktop/python stuffs/gui/write.png</normaloff>../home/shovon/Desktop/python stuffs/gui/write.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bttnread">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../home/shovon/Desktop/python stuffs/gui/book-open-icon.png</normaloff>../home/shovon/Desktop/python stuffs/gui/book-open-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bttnverify">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../home/shovon/Desktop/python stuffs/gui/Check-icon.png</normaloff>../home/shovon/Desktop/python stuffs/gui/Check-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="gpcs_chip">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../home/shovon/Desktop/python stuffs/gui/chipimage.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bttnerase">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase Chip</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../home/shovon/Desktop/python stuffs/gui/1414641771_DeleteRed.png</normaloff>../home/shovon/Desktop/python stuffs/gui/1414641771_DeleteRed.png</iconset>
    </property>
   </widget>
   <widget class="QCheckBox" name="chksetsck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Set slow SCK (USBasp only)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpfuse">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Fuse Settings</string>
    </property>
    <widget class="QLabel" name="lblpredef">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Saved Fuse Settings</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>hfuse:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>lfuse:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>efuse:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbopredefs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>221</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txthfuse">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtlfuse">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtefuse">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bttnclock">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Clock Speed...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bttnadvanced">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Advanced...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bttnwritefuse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Write Fuse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bttnreadfuse">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Read Fuse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bttncalnlock">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Cal. and Lock...</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bttneeprom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>EEPROM Editor...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../home/shovon/Desktop/python stuffs/gui/writing_app.png</normaloff>../home/shovon/Desktop/python stuffs/gui/writing_app.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lblchipname">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblnote">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click on the chip for&lt;/p&gt;&lt;p&gt;auto detection&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblstatus">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>331</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mnumenubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>694</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuProgrammers_Boards">
    <property name="title">
     <string>Programmers/Boards</string>
    </property>
    <widget class="QMenu" name="menuArduino_Boards">
     <property name="title">
      <string>Arduino Boards</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuOther_Custom">
     <property name="title">
      <string>Other/Custom</string>
     </property>
    </widget>
    <addaction name="actionUSBasp"/>
    <addaction name="actionAVRisp_2"/>
    <addaction name="menuArduino_Boards"/>
    <addaction name="menuOther_Custom"/>
    <addaction name="separator"/>
    <addaction name="actionManage"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuProgrammers_Boards"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionUSBasp">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>USBasp</string>
   </property>
  </action>
  <action name="actionManage">
   <property name="text">
    <string>Manage...</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actionAVRisp_2">
   <property name="text">
    <string>AVRisp/ArduinoISP</string>
   </property>
  </action>
  <action name="actionA">
   <property name="text">
    <string>a</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
